### PR TITLE
ANC Subsequent Visit - fixes 01

### PR DESCRIPTION
**Pull request by anvmn**
#252
#256
#257 
#279 (https://github.com/TIP-Global-Health/eheza-app/issues/279#issuecomment-1162871509 , found issue 1)
#228 
#242 
#237 
#232 
#230 
#231 
#240 
#245
#154

### CODE REVIEW

**Comment by anvmn** [comment]
Selection_875

**Comment by anvmn** [comment]
@mariano-dagostino please review

File: client/src/elm/Translate.elm [hunk 1]
```diff
@@ -8987,12 +8992,12 @@ translationSet trans =
 
                 EducationLowBackPain ->
                     if isInitial then
-                        { english = "Low back pain in pregancy - provided health education on " ++ date
+                        { english = "Lower backer pain in pregancy - provided health education on " ++ date
```
Comment (mariano-dagostino) [inline client/src/elm/Translate.elm]: @anvmn This sentence does not seems correct. Maybe `Lower back pain in pregnancy`?

File: client/src/elm/Translate.elm [hunk 2]
```diff
@@ -8987,12 +8992,12 @@ translationSet trans =
...
                         , kinyarwanda = Nothing
                         }
 
                     else
-                        { english = "Persistent low back pain in pregancy - provided health education on " ++ date
+                        { english = "Persistent lower back pain in pregancy - provided health education on " ++ date
```
Comment (mariano-dagostino) [inline client/src/elm/Translate.elm]: @anvmn pregnancy?

**Review by mariano-dagostino — APPROVED** [review]
@anvmn Looks good. Found 2 typos.

File: client/src/elm/Translate.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Translate.elm]: Thanks @mariano-dagostino

**Pull request merged**